### PR TITLE
[scatter] incorporate tree widget into scatter tool

### PR DESCRIPTION
**Pull request by chejennifer**
dev updated (https://dev.datacommons.org/tools/scatter)

### CODE REVIEW

File: static/js/tools/scatter/statvar.tsx [hunk 1]
```diff
@@ -33,28 +33,21 @@ import {
...
-} from "./context";
-import { NamedPlace } from "../../shared/types";
-import { nodeGetStatVar } from "./util";
+import { StatVarInfo, getStatVarInfo } from "../statvar_menu/util";
+import { Context, EmptyAxis, Axis, AxisWrapper } from "./context";
+import { StatVarHierarchyType } from "../../shared/types";
+import { StatVarHierarchy } from "../../stat_var_hierarchy/stat_var_hierarchy";
 
-interface NamedStatVar {
+const SAMPLE_SIZE = 3;
```
Comment (beets) [inline static/js/tools/scatter/statvar.tsx:41]: add a short description for this - what is being sampled?

**Review by beets — APPROVED** [review]
nice!

File: static/js/tools/scatter/statvar.tsx (hunk 1 above)
Comment (chejennifer) [reply static/js/tools/scatter/statvar.tsx:43]: done

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Pull request merged**